lib/optimize/SplitChunksPlugin.js

```diff
@@ -28,10 +28,11 @@ const MinMaxSizeWarning = require("./MinMaxSizeWarning");
 
 /**
  * @typedef {Object} ChunksInfoItem
- * @property {SortableSet<Module>} modules
+ * @property {SortableSet} modules
```

Comment:
Why did you changed that?

Reply:
I moved the duplicate typedef from below. I'll restore it.